Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/QuadLight.azsli

```diff
@@ -148,13 +148,14 @@ void ApplyQuadLight(ViewSrg::QuadLight light, Surface surface, inout LightingDat
                 GetDiffuseLighting(surface, lightingData, intensity, dirToLightCenter)
             );
             
+            float nullTransmissionDistance = (o_transmission_mode == TransmissionMode::ThickObject) ? 0.0f : 9999.0f;
```

Comment:
Would it be better to simply add that constant within `GetBackLighting` by checking the transmission mode there instead of needing to perform this initialization for all light types?

Reply:
You are right... I've changed this so that instead of initializing checking to the transmission mode, it initializes to `transmissionDistance = -1.0f`. Then, if shadows are not enabled, the variable won't be redefined and the `TransmissionMode::ThinObject` case in the `GetBackLighting` function will detect if `transmissionDistance < 0.0f`, and avoid computing the transmission in that case.
`ThickObject` transmission can be applied with shadows disabled, and in that case it does not matter if we use a 'null' value of 0.0f or -1.0f because the line `thickness = max(transmissionDistance, surface.transmission.thickness);` in the `GetBackLighting` `TransmissionMode::ThickObject` case will return the same (`surface.transmission.thickness` range is [0,1]).